test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,35 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
+        // has null objects
+        assertHasNull((Object) null);
```

Comment:
To make this test case clearer, maybe you can try further separating the case into `// has single null objects` & `// has multiple null objects`